doc/source/manual/xtrabackup_sst.rst

```diff
@@ -338,9 +338,13 @@ To disable initial SST timeout, set ``sst-initial-timeout=0``.
    :Example: /path/to/tmp/dir
 
 This option specifies the location for storing the temporary file
-where the transaction log is stored
+on a donor node where the transaction log is stored
 before streaming or copying it to a remote host.
 
+.. Note:: Starting from |PXC| 5.7.20-29.24 this option can be used on
+   joiner node also, to specify location for the received SST file other
+   than *datadir/.sst*.
+
```

Comment:
Maybe something like this would be better (this is temporary storage and must be large enough)
=====

to specify a location to receive temporary SST files other than *datadir/.sst*.  This location must be large enough to hold the contents of the entire database.

Also, if tmpdir is empty then *datadir/.sst* will be used.